scaffolds/csharp/docker.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker run -it --rm -v $PWD:/app -w /app/src/CodeRetreat microsoft/dotnet:1.1-sdk-projectjson bash
+docker run -it --rm -v ${PWD}:/app -w /app/CodeRetreat.Tests microsoft/dotnet:2.0.0-sdk bash
```

Comment:
This is a good opportunity to add a `$@` at the end of the line so that arguments are passed to the bash command :)

Reply:
Fixed.